lib/screen/home_screen.dart

```diff
@@ -145,10 +146,13 @@ class _HomeScreenState extends State<HomeScreen> with CommandHandler {
 
   /// Create notes query
   Stream<List<Book>> _createBookStream(BuildContext context) {
-    final user = Provider.of<CurrentUser>(context)?.data;
+    final user = FirebaseAuth.instance.currentUser;
     final collection = booksCollection(user?.uid);
 
     return collection
+        .where('state',
+        isLessThan: BookState.deleted
+            .index)
```

Comment:
I'm a bit curious about this. Is `deleted` a number or a boolean? Because `isLessThan` makes me think it's a boolean.

Reply:
It is a number, in fact we store the index of the `enum`, so we store an `integer`. The `isLessThan` means that we are making sure that we get anything less than the delete index which in fact is the active index. Firestore don't have lot of filter method